js/grumbles/form.directive.js

```diff
@@ -21,9 +21,13 @@
       },
       link: function(scope, element, attributes){
         if(!scope.grumble) scope.grumble = new GrumbleFactory();
-        scope.ifSaveSuccessful = function(response){
-          console.log(response);
-          $state.go($state.current, {}, {reload: true});
+        scope.ifCreateSuccessful = function(response){
+          scope.$parent.GrumbleIndexViewModel.grumbles.push(response);
```

Comment:
what does `$parent` do here? I like the change to just adding items to array rather than forcing a state change, but does this affect anything else in the code?